kbase-extension/static/kbase/js/widgets/narrative_core/upload/stagingAreaViewer.js

```diff
@@ -430,8 +440,8 @@ define([
                         }
                     });
 
-                    $('td:eq(1)', nRow).find('button[data-decompress]').off('click');
-                    $('td:eq(1)', nRow).find('button[data-decompress]').on('click', e => {
+                    $('td:eq(2)', nRow).find('button[data-decompress]').off('click');
+                    $('td:eq(2)', nRow).find('button[data-decompress]').on('click', e => {
```

Comment:
Pre-existing code, I know, but `$('td:eq(2)', nRow).find('button[data-decompress]')` could be saved to a variable, or you can probably do
```js
$('td:eq(2)', nRow).find('button[data-decompress]')
  .off('click')
  .on('click', e => { ... })
```
to save the extra DOM call

Reply:
aha! I scroll up and see someone had the same thought as me!